TACC/Core-Portal
# FP-589: Add workPath to job modal

## Overview: ##

Add workPath to job modal

## PR Status: ##


## Related Jira tickets: ##

* [FP-589](https://jira.tacc.utexas.edu/browse/FP-589)

## Summary of Changes: ##

## Testing Steps: ##
1. Load job modal
2. Check that Temporary Working Directory is in the configuration section

## UI Photos:

![Screen Shot 2020-08-18 at 8 01 50 PM](https://user-images.githubusercontent.com/8287580/90580319-288a9d80-e18e-11ea-834b-57c3c58a55bd.png)
